CHANGELOG.md

```diff
@@ -7,16 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2022-03-30
 
+Amazon Chime SDK JS v3 is here !! 🎉🎉🎉
+
+Amazon Chime SDK for JavaScript v3 includes major improvements for device management, WebRTC metrics, and the 
+messaging session.
+
+- **Device management:** Decouple each user’s audio and video devices from their connection to the WebRTC media 
+  session. A user can select their preferred devices in a preview window prior to joining the session, and the devices do not 
+  have to be programmatically reselected when joining the session. Once connected to the session, the user can switch devices instantly without interrupting their WebRTC connection.
+- **WebRTC metrics:** Publish the standardized WebRTC metrics for all supported browsers.
+- **Messaging session:** Add support for AWS SDK for JavaScript v3 for messaging session to help avoid using multiple 
```

Comment:
> Support AWS SDK for JavaScript v3 in the messaging session.

(`To help avoid using multiple versions ...` part does not make sense for AWS SDK v2 users?)